src/core_plugins/kibana/public/dashboard/dashboard_strings.js

```diff
@@ -28,10 +29,26 @@ import { DashboardViewMode } from './dashboard_view_mode';
  */
 export function getDashboardTitle(title, viewMode, isDirty) {
   const isEditMode = viewMode === DashboardViewMode.EDIT;
-  const unsavedSuffix = isEditMode && isDirty
-    ? ' (unsaved)'
-    : '';
+  let displayTitle;
 
-  const displayTitle = `${title}${unsavedSuffix}`;
-  return isEditMode ? 'Editing ' + displayTitle : displayTitle;
+  if (isEditMode && isDirty && isEditMode) {
```

Comment:
Duplication of `isEditMode` in this condition.
I think it can be 
```
if (isEditMode && isDirty) {
    displayTitle = i18n.translate('kbn.dashboard.strings.dashboardUnsavedEditTitle', {
      defaultMessage: 'Editing {title} (unsaved)',
      title,
    });
  } else if (isEditMode) {
      displayTitle = i18n.translate('kbn.dashboard.strings.dashboardEditTitle', {
        defaultMessage: 'Editing {title}',
        title,
      });
  } else {
    displayTitle = title;
  }
```